istioctl/pkg/writer/pilot/status.go

```diff
@@ -103,5 +106,17 @@ func xdsStatus(sent, acked string) string {
 	if sent == acked {
 		return "SYNCED"
 	}
-	return "STALE"
+	timeSent, _ := parseTime(sent)
+	timeAcked, _ := parseTime(acked)
+	timeDiff := timeAcked.Sub(timeSent)
+	var buffer bytes.Buffer
+	buffer.WriteString("STALE")
+	buffer.WriteString(timeDiff.String())
+	return buffer.String()
+}
+
+func parseTime(s string) (time.Time, error) {
+	i, err := strconv.ParseInt(s, 10, 64)
+	t := time.Unix(i, 0)
```

Comment:
Does this work? The nonces look like this `2009-11-10 23:00:00 +0000 UTC m=+0.000000001`.